solr/core/src/test/org/apache/solr/search/TestCoordinatorRole.java

```diff
@@ -296,6 +294,8 @@ public void testNRTRestart() throws Exception {
         assertEquals(nrtCore, hostCore);
         // allow any exceptions to propagate
         jettyManipulationFuture.get();
+
+        // TODO what is expected here? why are we just bailing? Can't the following code just be removed...
```

Comment:
this got accidentally added while debugging. this line must go

Reply:
Removed in 869ee24dcfb1fbed253291d2f49945d186eaccda